packages/pwa-kit-cli/bin/cli.js

```diff
@@ -126,6 +126,16 @@ const main = () => {
                     ...process.env
                 }
             })
+
+            // Copy the project `package.json` into the build folder.
+            sh.cp(p.join(process.cwd(), 'package.json'), './build/package.json')
+
+            // Copy config files.
+            const config = p.join(process.cwd(), 'config')
```

Comment:
```suggestion
            const config = p.resolve('config')
```
Relative paths are usually assumed to be relative to `process.cwd()`, so is the full path actually necessary here?